Run/CMakeLists.txt

```
@@ -4,4 +4,4 @@
 #
 
 add_run(EnergyPositionHistos EnergyPositionHistosRun.cpp)
-add_run(TruthTracking TruthTrackingRun.cpp)
+#add_run(TruthTracking TruthTrackingRun.cpp)
```

```suggestion
add_run(TruthTracking TruthTrackingRun.cpp)
```
Let's keep changes like that to local. Also, we need to check if they fixed the dictionary issue